c_tests/tests/const_global.c

```diff
@@ -8,7 +8,7 @@
 //     jit-state: stop-tracing
 //     --- Begin jit-pre-opt ---
 //     ...
-//     define internal void @__yk_compiled_trace_0(%YkCtrlPointVars* %0) {
+//     define internal void @__yk_compiled_trace_0(%YkCtrlPointVars* %0, i64* %1, i64 %2) {
```

Comment:
remind me what the extra trace args are again.

Reply:
It's explained further down in `jitmodbuilder`. These are the address and size of the stackmap.